### PR TITLE
Crashes android last session crash report

### PR DESCRIPTION
**Pull request by hrafnkellbaldurs**
* Added android implementation for `Crashes.lastSessionCrashReport`
* Updated example to display the crash report

### CODE REVIEW

File: appcenter-crashes/android/src/main/java/com/getcapacitor/plugin/appcenter/crashes/CrashesBase.java [hunk 1]
```diff
@@ -1,8 +1,11 @@
 package com.getcapacitor.plugin.appcenter.crashes;
 
+import com.getcapacitor.JSObject;
 import com.microsoft.appcenter.AppCenter;
 import com.microsoft.appcenter.crashes.Crashes;
 
+import java.util.Map;
```
Comment (johnborges) [inline appcenter-crashes/android/src/main/java/com/getcapacitor/plugin/appcenter/crashes/CrashesBase.java:7]: Is this needed?

Comment (hrafnkellbaldurs) [reply appcenter-crashes/android/src/main/java/com/getcapacitor/plugin/appcenter/crashes/CrashesBase.java]: Good catch 😄 No it's not needed, I've removed it now.

**Comment by hrafnkellbaldurs** [comment]
Should be all done now 👍

**Comment by johnborges** [comment]
@all-contributors please add @hrafnkellbaldurs for code

**Pull request merged**